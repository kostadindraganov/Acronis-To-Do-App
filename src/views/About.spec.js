import { createLocalVue, mount } from "@vue/test-utils";
import ElementUI from "element-ui";
import About from "@/views/About";

const localVue = createLocalVue();
localVue.use(ElementUI);

describe("About.vue", () => {
	test("renders correctly", () => {
		const wrapper = mount(About, {
			localVue
		});
		expect(wrapper.element).toMatchSnapshot();
	});
});
