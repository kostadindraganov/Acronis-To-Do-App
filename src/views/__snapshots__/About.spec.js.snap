// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue renders correctly 1`] = `
<div
  class="el-row is-justify-center is-align-middle el-row--flex"
>
  <div
    class="el-col el-col-24 el-col-xs-24 el-col-sm-22 el-col-md-20 el-col-lg-20 el-col-xl-22"
  >
    <div
      class="el-card box-card is-always-shadow"
    >
      <!---->
      <div
        class="el-card__body"
      >
        <div
          class="el-row is-justify-space-around is-align-middle el-row--flex"
          style="margin-left: -15px; margin-right: -15px;"
        >
          <div
            class="el-col el-col-22"
            style="padding-left: 15px; padding-right: 15px;"
          >
            <h2>
              Why Use a To-Do List?
            </h2>
             
            <p>
              We all have those days when there are a million things to do, and we don't know how we're going to get it all done. It's easy to become overwhelmed by the vast quantity of tasks that we must do from day to day. Sometimes we have so many balls in the air, that we may even lose track of some loose ends and forget to do important things. When we get too busy, we end up feeling like we are barely able to keep our heads above water.
            </p>
             
            <h3>
              Organization
            </h3>
             
            <p>
              
            One of the most important reasons for keeping a to-do list is the organization. Organizing your tasks with a list can make everything much more manageable and make you feel grounded. Seeing a clear outline of your completed and uncompleted tasks will help you feel organized and stay mentally focused.
            As you cross items off your to-do list, you'll feel a sense of progress and accomplishment that can be missed when rushing from one activity to the next. The affirmation that you are making progress will help motivate you to keep moving forward rather than feeling overwhelmed.
            Having a list of all your tasks will allow you to sit down and make a plan. One study showed that fifteen minutes spent planning could save an hour of execution time!
          
            </p>
             
            <h3>
              Productivity
            </h3>
             
            <p>
              If you record all your tasks in a to-do list, you can easily review the list and prioritize the most important tasks. Why waste time on trivial activities when there are important matters that need your attention? One study, conducted by the Harvard Business Review, showed that 90% of managers wasted valuable time through poor time management. Your to-do list will help you focus your attention on the most important task of the moment.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
